packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -14,6 +14,7 @@ import { Icon, Box, Label, Text } from 'gestalt';
 import PropTypes from 'prop-types';
 import DatePickerTextField from './DatePickerTextField.js';
 import styles from './DatePicker.css';
+import format from './format.js';
```

Comment:
@AlbertCarreras could we rename this to `dateFormat`?

Reply:
DONE